src/modules/editTask.js

```diff
@@ -0,0 +1,104 @@
+const todoList = document.querySelector('.todo-list');
+
+const runComplete = (inputEl) => {
+  const localData = JSON.parse(localStorage.getItem('tasks'));
+  localData.forEach((task) => {
+    if (inputEl.checked !== false) {
+      if (Number(inputEl.id) === Number(task.index)) {
+        task.completed = true;
+      }
+    } else {
+      task.completed = false;
+    }
+    localStorage.setItem('tasks', JSON.stringify(localData));
+  });
```

Comment:
As the previous reviewer suggested, your app is not working entirely as expected 👇 

- [ ] Every other to-do item gets edited when I try to edit a single to-do item. Please fix it
- [ ] When adding a new to-do item, it duplicates the existing data and then proceeds to refresh the page
- [ ] After marking a task as completed, it doesn't remain so when I reload the page
**Please implement these suggestions**